anti-corruption-layer/src/main/java/com/iluwatar/corruption/system/modern/Customer.java

```diff
@@ -0,0 +1,50 @@
+package com.iluwatar.corruption.system.modern;
+
+import java.util.Objects;
+
+/**
+ * The class represents a customer in the modern system.
+ */
+public class Customer {
```

Comment:
Use Lombok to get rid of boilerplate such as getters, setters, constructors, equals, hashCode, etc.